scripts/lint-bib.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+bibtool --preserve.key.case=on --preserve.keys=on --pass.comments=on --print.use.tab=off -s \
+  -i docs/references.bib -o docs/references.bib
+git diff --exit-code
```

Comment:
I think it might be slightly more robust to do something like (untested!)
```suggestion
cp docs/references.bib docs/references.bib.old
bibtool --preserve.key.case=on --preserve.keys=on --pass.comments=on --print.use.tab=off -s \
  -i docs/references.bib -o docs/references.bib
diff docs/references.bib.old docs/references.bib 
```
since then a user could run this on a local copy and not get a surprising exit code if they have uncommitted changes.